src/dbt_score/models.py

```diff
@@ -661,18 +670,22 @@ def _reindex_tests(self) -> None:
                 ):
                     self.tests[node_unique_id].append(node_values)
 
-    def _populate_parents(self) -> None:
-        """Populate `parents` for all models and snapshots."""
+    def _populate_relatives(self) -> None:
+        """Populate `parents` and `children` for all models, sources and snapshots."""
```

Comment:
```suggestion
        """Populate `parents` and `children` for all dbt entities."""
```

it was missing `seeds` :grin: